### PR TITLE
Alt Title Bloat

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

Ports new titles from Nova, adds a variety of new ones. I went a bit insane.

## How This Contributes To The Skyrat Roleplay Experience

More flavor.

## Proof of Testing

It works.

## Changelog
:cl:
add: Even more alt titles, God.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 1]
```diff
@@ -19,17 +19,32 @@
 /datum/job/assistant
 	alt_titles = list(
 		"Assistant",
+		"Ambassador"
```
Comment (theselfish) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]:
```suggestion
		"Ambassador",
```

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 2]
```diff
@@ -19,17 +19,32 @@
...
 		"Artist",
+		"Actor",
 		"Businessman",
 		"Businesswoman",
+		"Butler",
+		"Colonist",
+		"Contractor",
 		"Civilian",
+		"Duke",
+		"Diplomat"
```
Comment (theselfish) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]:
```suggestion
		"Diplomat",
```

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 3]
```diff
@@ -177,11 +215,15 @@
 /datum/job/cook
 	alt_titles = list(
 		"Cook",
+		"Baker"
```
Comment (theselfish) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]:
```suggestion
		"Baker",
```

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 4]
```diff
@@ -424,8 +503,13 @@
 /datum/job/warden
 	alt_titles = list(
 		"Warden",
+		"Armory Superintendent"
```
Comment (theselfish) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]:
```suggestion
		"Armory Superintendent",
```

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 5]
```diff
@@ -424,8 +503,13 @@
 /datum/job/warden
 	alt_titles = list(
 		"Warden",
+		"Armory Superintendent",
 		"Brig Sergeant",
+		"Brig Sentry",
 		"Brig Governor",
+		"Deputy Commissioner of Security",
 		"Dispatch Officer",
+		"Deputy Commissioner
```
Comment (theselfish) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]:
```suggestion
		"Deputy Commissioner",
```

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 6]
```diff
@@ -19,17 +19,32 @@
...
+		"Ambassador",
 		"Artist",
+		"Actor",
 		"Businessman",
 		"Businesswoman",
+		"Butler",
+		"Colonist",
+		"Contractor",
 		"Civilian",
+		"Duke",
```
Comment (SpaceLoveSs13) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:31]: An assistant with the title duke seems very out of place on a space station.

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 7]
```diff
@@ -67,13 +85,15 @@
...
 		"Junior Runner",
+		"Professional Gamer",
 	)
 
 /datum/job/blueshield
 	alt_titles = list(
 		"Blueshield",
 		"Command Bodyguard",
 		"Executive Protection Agent",
+		"Command Goon",
```
Comment (SpaceLoveSs13) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: Is having the word "goon" really a good idea for all the guard roles?

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 8]
```diff
@@ -122,29 +145,43 @@
...
 		"Reverend",
-		"Cleric",
+		"Shrine Maiden",
+		"Shrine Guardian",
 	)
 
 /datum/job/chemist
 	alt_titles = list(
 		"Chemist",
+		"Alchemist",
```
Comment (SpaceLoveSs13) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:182]: fix the L

Comment (theselfish) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:182]: https://www.merriam-webster.com/dictionary/alchemist

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm (hunk 7 above)
Comment (theselfish) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: Ehh, I wasn't sold on it, I'll axe it.

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm (hunk 6 above)
Comment (theselfish) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:31]: I thought king was too high, I wanted something for minor noble gimmicks.

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm (hunk 8 above)
Comment (SpaceLoveSs13) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:182]: > https://www.merriam-webster.com/dictionary/alchemist

Ik what alchemist is, I didnt realize github was gaslighting me into thinking that is a '1' after the A.

Comment (theselfish) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:182]: Oh lol.

**Comment by SpaceLoveSs13** [comment]
you can keep the duke, it's fine.

**Comment by theselfish** [comment]
Cool.

**Pull request merged**